### ISSUE TITLE
Setup doesn't work with `attw`

### ISSUE THREAD

**Issue by SBoudrias** [waldekmastykarz/node-ts-cjs-esm#1]
This setup has a few issues:

1. It breaks under this condition https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/UntypedResolution.md
2. If there's more than a single file, the `.sh` script will not properly update ESM import paths.

**Comment by waldekmastykarz** [comment]
Good catch! Do you have a proposal for fixing 1? Would you like to submit a PR for 2?

**Comment by waldekmastykarz** [comment]
Closing due to lack of response
